packages/combobox/src/Combobox.types.ts

```diff
@@ -277,19 +277,23 @@ export type ComboboxProps<M extends boolean> = Either<
  * Combobox Option Props
  */
 
-interface BaseComboboxOptionProps extends ComponentPropsWithoutRef<'li'> {
+type LIProps = Omit<ComponentPropsWithoutRef<'li'>, 'onClick'>;
```

Comment:
nit - Should the `LI` be all caps? I think something like `ListItemProps` would be more legible here